db/migrate/20170110145641_add_host_action_button_to_remote_execution_feature.rb

```diff
@@ -0,0 +1,5 @@
+class AddHostActionButtonToRemoteExecutionFeature < ActiveRecord::Migration
+  def change
+    add_column :remote_execution_features, :host_action_button, :bool, :null => false, :default => false
```

Comment:
I've noticed this in schema.rb

```
# Could not dump table "remote_execution_features" because of following NoMethodError
# undefined method `[]' for nil:NilClass
```

This meant `bundle exec rake db:schema:load RAILS_ENV=test` was not populating the `:remote_execution_features` table. Changing `:bool` to `:boolean` seemed to do the trick